server/neptune/workflows/internal/deployment/deployment.go

```diff
@@ -0,0 +1,24 @@
+package deployment
```

Comment:
Why are we creating a new package just for one file? This seems a bit overkill, i noticed this happened with `root` and `job` as well.  Packages exist to group related functions, if we get so granular with it where theres a bunch of packages with one file or one struct, i think it's time to rethink the structure.

Reply:
Yeah right, I moved deployment.go file inside the root package!